docs/src/Sheet.doc.js

```diff
@@ -307,27 +377,27 @@ function SheetRefExample() {
       <Button
         inline
         text="Open sheet"
-        onClick={() => { setShowSheet(!showSheet) }}
+        onClick={() => setShouldShow(true)}
       />
-      {showSheet && (
+      {shouldShow && (
         <Layer zIndex={sheetZIndex}>
           <>
             <Sheet
               accessibilityDismissButtonLabel="Close"
               accessibilitySheetLabel="Focused sheet"
-              onDismiss={() => { setShowSheet(!showSheet) }}
+              onDismiss={() => setShouldShow(false)}
               ref={sheetRef}
               size="md"
             >
               <Box color="white" minHeight={400} padding={8}>
                 <Box marginBottom={4}>
-                  <Heading size="md">Focused content</Heading>
+                  <Heading size="md">Focused content</Heading>                
```

Comment:
@themindfuldev looks like it's adding a bunch of extra whitespace after certain lines, can we remove that?
![image](https://user-images.githubusercontent.com/127199/95138003-194fb700-071e-11eb-9d58-4830444c3ba1.png)

Reply:
Sure - removing as part of https://github.com/pinterest/gestalt/pull/1242